### ISSUE TITLE
Website Accessibility

### ISSUE THREAD

**Issue by joelachance** [numpy/numpy.org#118]
We need to check that the new website can be navigated with browser accessibility tools.

- [x] Test happy paths and make sure there aren't any bugs when navigating with accessibility tools.

**Comment by MarsBarLee** [comment]
As suggested in a previous PR involving web-accessibility, I'll look into adding aria-labels.

**Comment by joelachance** [comment]
Some additional changes on branch: feature/118-accessibility

**Comment by joelachance** [comment]
@MarsBarLee @rgommers @InessaPawson 

I just did a quick once-over with Mac's VoiceOver and things look pretty good. CodeMirror (our interactive shell) isn't set up very well for accessibility, but I'm not sure there's much we can do there. Overall, I was able to navigate the site without any major issues.

One callout is that the install instructions aren't called out as code blocks in VoiceOver, but since those are being rendered using some Hugo magic, I'm not sure we can tag those with aria labels.
